invenio_records_resources/services/references/schema.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 TU Wien.
+#
+# Invenio-Records-Resources is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Schema for entity references."""
+
+from marshmallow import RAISE, Schema, ValidationError, fields, \
+    validates_schema
+
+
+#
+# Schema for entity references
+#
+class EntityReferenceBaseSchema(Schema):
+    """Base schema for entity references, allowing only a single key.
+
+    Example of an allowed value: ``{"user": 1}``.
+    Example of a disallowed value: ``{"user": 1, "record": "abcd-1234"}``.
+    """
+
+    class Meta:
+        """Schema meta."""
+
+        unknown = RAISE
```

Comment:
This is the default, so we don't have to specify it anymore: https://marshmallow.readthedocs.io/en/stable/quickstart.html#handling-unknown-fields  .